distribution/test/itests/test-itests-catalog/src/test/java/ddf/catalog/test/TestFederation.java

```diff
@@ -217,13 +330,20 @@ public void testFederatedRetrieveExistingProductCsw() throws Exception {
      */
     @Test
     public void testFederatedRetrieveNoProduct() throws Exception {
+        // Setup 
+        urlResourceReaderConfigurator.setUrlResourceReaderRootDirs(new String[] {DEFAULT_URL_RESOURCE_READER_ROOT_RESOURCE_DIRS});        
         String restUrl = REST_PATH + "sources/" + OPENSEARCH_SOURCE_ID + "/"
                 + metacardIds[GEOJSON_RECORD_INDEX] + "?transform=resource";
+        
+        // Perform Test and Verify
         when().get(restUrl).then().log().all().assertThat().statusCode(equalTo(500));
     }
 
     @Test
     public void testFederatedRetrieveNoProductCsw() throws Exception {
+        File[] rootDirectories = File.listRoots();        
+        String rootDir = rootDirectories[0].getCanonicalPath();
+        urlResourceReaderConfigurator.setUrlResourceReaderRootDirs(new String[] {rootDir});
```

Comment:
If you do not make KARAF_HOME the default, you could add a method like useKarafHome to simplify configuring this.